hedyorg/hedy
# 🧹 Remove debug prints that were left in and fix debug logging

The user activity tracking code logs a lot of errors to the JavaScript console. Nothing goes wrong, but the `postJson` function that is begin used expects a `200` response, but is getting a `204` response.

Only signal an error if we actually get a `400` or higher response.

Also remove some Python `print()`s that were left in most likely for debugging.

**How to test**

Open the console, use the website, observe that the following error does not appear:

![image](https://github.com/hedyorg/hedy/assets/524162/2e810a6e-009a-4f6d-98f5-d44dcb930c84)
